src/bitcoin-cli.cpp

```diff
@@ -992,8 +993,18 @@ static void ParseGetInfoResult(UniValue& result)
         RESET);
     result_string += strprintf("Version: %s\n", result["version"].getValStr());
     result_string += strprintf("Time offset (s): %s\n", result["timeoffset"].getValStr());
-    const std::string proxy = result["proxy"].getValStr();
-    result_string += strprintf("Proxy: %s\n", proxy.empty() ? "N/A" : proxy);
+
+    std::set<std::string> unique_proxies;
+
+    for (const UniValue& network : result["networks"].getValues()) {
+        unique_proxies.insert(network["proxy"].getValStr());
+    }
+
+    std::string proxies = std::accumulate(unique_proxies.begin(), unique_proxies.end(), std::string{}, [](const std::string& a, const std::string& b) {
+        return a.empty() ? b : a + ", " + b;
+    });
```

Comment:
I think you can use our `Join` helper from `src/util/string.h` to achieve the same (this only works on a std::vector though, so you'd need to convert first):
```suggestion
    std::string proxies = Join(unique_proxies_vector, ", ");
```

Reply:
Thanks for the suggestion! Updated PR use `Join` instead.